hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -40,6 +40,11 @@
     private String fullPath;
 
     /**
+     * Relative path to the file from the base path
+     */
+    private String relativePath;
```

Comment:
@prazanna need your sign off here to verify this wont break the incremental pipelines currently live